src/js/models/list-section.js

```diff
@@ -0,0 +1,68 @@
+import Section from './markup-section';
+import LinkedList from '../utils/linked-list';
+
+export const LIST_SECTION_TYPE = 'list-section';
+
+export default class ListSection extends Section {
+  constructor(tagName, items=[]) {
+    super(tagName);
+    this.type = LIST_SECTION_TYPE;
+
+    // remove the inherited `markers` because they do nothing on a ListSection but confuse
+    this.markers = undefined;
```

Comment:
mmhm. Right. I almost think there is an alternative strategy of having `ListItem` and `MarkupSection` extend from `Markerable` instead of having `ListSection` inherit from `MarkupSection`. The bulk of the shared behaviors are probably around markers.